.ci/test-run-scheduler-schema.sql

```diff
@@ -0,0 +1,148 @@
+create type test_run_status as enum ('running', 'passed', 'failed');
+
+create table busted_test_run_request
+(
+    id           serial primary key,
+    workflow_id  bigint    not null,
+    created_at   timestamp not null default current_timestamp,
+    test_file    varchar   not null,
+    suite        varchar   not null,
+    exclude_tags varchar   not null,
+    environment  varchar   not null
```

Comment:
```suggestion
    test_file    text   not null,
    suite        text   not null,
    exclude_tags text   not null,
    environment  text   not null
```

(same but shorter, we in general use text in kong)

Reply:
While technically the same, `varchar` is more indicative of a single line whereas `text`, at least to me, is multiline text.  Thus, I'd like to keep it this way as a form of documentation.